string header
uint32[] scan_id
uint32[] point_id
time[] stamp
float32[] unflatness
float32[] angle
float32[] score
